tensorboard/plugins/core/core_plugin_test.py

```diff
@@ -231,6 +231,16 @@ def _generate_test_data(self, run_name):
     writer.flush()
     writer.close()
 
+    # Write data for the run to the database.
+    tf.reset_default_graph()
```

Comment:
Can you add a TODO(nickfelt) here to figure out why this is required, i.e. why we have to reset the graph?

Reply:
Done. Thanks for looking into this and starting a discussion.